ci/scripts/integration_spark.sh

```diff
@@ -35,7 +39,7 @@ fi
 export MAVEN_OPTS="-Xss256m -Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
 export MAVEN_OPTS="${MAVEN_OPTS} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
 
-pushd ${spark_dir}
+pushd "${spark_dir}"
   echo "Building Spark ${SPARK_VERSION}"
```

Comment:
We can use `spark_version` not `SPARK_VERSION`.

Reply:
All  of `SPARK_VERSION` changed to `spark_version`.